lib/pure/terminal.nim

```diff
@@ -727,6 +727,14 @@ proc getch*(): char =
       doAssert(readConsoleInput(fd, addr(keyEvent), 1, addr(numRead)) != 0)
       if numRead == 0 or keyEvent.eventType != 1 or keyEvent.bKeyDown == 0:
         continue
+      case keyEvent.uChar
+      of 0, 0x0E:
```

Comment:
What's 0x0E?

Reply:
Actually this was me being dyslexic - good of you to question it ;)- I actually saw 0xE0 (Extended key) and not 0x0E (shift out). I've been doing some testing and I never received the extended key in KEY_EVENT_RECORD .uChar which means the test can be simplified - I've updated the pull request accordingly.